baselines/dom.generated.d.ts

```diff
@@ -8249,20 +8249,20 @@ interface Node extends EventTarget {
     readonly previousSibling: Node | null;
     textContent: string | null;
     appendChild<T extends Node>(newChild: T): T;
-    cloneNode(deep?: boolean): Node;
+    cloneNode(deep?: boolean): this;
```

Comment:
This will make all the Node hierarchy generic, something we have avoided for perf reasons. see https://github.com/Microsoft/TSJS-lib-generator/pull/166 for more details.

The alternative here is for every interface that extends from Node, we can redefine `cloneNode` to return the containing type.